experiments/gail_benchmark.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+
+# This script finds the the mean and standard error of episode return after
+# training GAIL on benchmark tasks.
+#
+# The CSV can be modified and regenerated from following spreadsheet:
```

Comment:
Explain this is a CSV config. At first I assumed it was output and was confused.